lib/srv/forward/sshserver.go

```diff
@@ -646,7 +646,7 @@ func (s *Server) handleChannel(nch ssh.NewChannel) {
 func (s *Server) handleDirectTCPIPRequest(ch ssh.Channel, req *sshutils.DirectTCPIPReq) {
 	// Create context for this channel. This context will be closed when
 	// forwarding is complete.
-	ctx, err := srv.NewServerContext(s, s.sconn, s.identityContext)
+	ctx, err := srv.NewServerContext(s, s.sconn, s.identityContext, nil)
```

Comment:
Can you add a comment explaining why you don't need a `ConnectionContext` here? Also, this means that Control Master works with the forwarding servers right?